railties/lib/rails/generators/rails/app/templates/Dockerfile.tt

```diff
@@ -33,13 +40,13 @@ RUN curl -sL https://github.com/nodenv/node-build/archive/master.tar.gz | tar xz
 <% end -%>
 # Install application gems
 COPY Gemfile Gemfile.lock ./
-RUN bundle install<% if depend_on_bootsnap? -%> && \
+RUN bundle _${BUNDLER_VERSION}_ install<% if depend_on_bootsnap? -%> && \
```

Comment:
I don't think you need that. Bundler automatically install the right version a re-exec itself.

That makes me think we should probably set `BUNDLE_DEPLOYMENT=1` though

Reply:
Bundler will automatically install the right version of itself... on each deploy.  Installing the right version once (in a prior Docker layer) will eliminate this.

Reply:
That's a very negligible overhead to install one gem when you are about to install a hundred. Not worth the downsides IMO.